src/js/components/Select/utils.js

```diff
@@ -7,3 +9,96 @@ export const applyKey = (option, key) => {
     return option[Object.keys(option)[0]];
   return option;
 };
+
+export const getOptionLabel = (index, options, labelKey) =>
+  applyKey(options[index], labelKey);
+
+export const getOptionValue = (index, options, valueKey) =>
+  applyKey(options[index], valueKey);
+
+export const checkDisabled = (
```

Comment:
If you made this a custom hook, it might make its usage cleaner:
```
const isDisabled = useDisabled(disabled, disabledKey, options, valueKey);
...
if (isDisabled(nextActiveIndex) ...
```
where `useDisabled` would then simply be:
```
const useDisabled(disabled, disabledKey, options, valueKey) = useCallback((index) => {...}, [disabled, disabledKey, options, valueKey]);
```

Reply:
There's just a lot of lines of code for dependencies in the other useCallbacks() in SelectContainer.js

Reply:
Changed to use `useDisabled` hook